zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/socket/Message.java

```diff
@@ -128,6 +128,7 @@
 
     LIST_NOTEBOOK_JOBS,     // [c-s] get notebook job management infomations
     LIST_UPDATE_NOTEBOOK_JOBS, // [c-s] get job management informations for until unixtime
+    UNSUBSCRIBE_UPDATE_NOTEBOOK_JOBS, // [c-s] unsubscribe job information for job management
                                // @param unixTime
```

Comment:
Could you remove "@param" section from the comment while `UNSUBSCRIBE_UPDATE_NOTEBOOK_JOBS` doesn't look like taking param.